apps/backend/src/drink-action/dto/update-drink-action.dto.ts

```diff
@@ -0,0 +1,5 @@
+import { PartialType } from '@nestjs/swagger';
+
+import { CreateDrinkActionDto } from './create-drink-action.dto';
+
+export class UpdateDrinkActionDto extends PartialType(CreateDrinkActionDto) {}
```

Comment:
drinkId and eventId should not be updateable, so remove those from this class with OmitType